src/shared/views/components/Spinner.tsx

```diff
@@ -1,5 +1,5 @@
-import React from 'react';
+import React, { CSSProperties } from 'react';
 
-export const Spinner = () => {
-  return <div className="loader" aria-live="polite" role="status"></div>;
+export const Spinner = (style?: CSSProperties) => {
+  return <div className="loader" aria-live="polite" role="status" style={style}></div>;
```

Comment:
Pass in an optional css class, not inline styles.

See https://github.com/Marvell-Consulting/statswales-frontend/blob/main/src/shared/views/components/Select.tsx#L34